src/res_grammar.ml

```diff
@@ -336,6 +338,7 @@ let isListElement grammar token =
   | JsxAttribute -> isJsxAttributeStart token
   | JsFfiImport -> isJsFfiImportStart token
   | AttributePayload -> token = Lparen
+  | TagNames -> token = Hash
```

Comment:
This is not currently used, as when returning `None` the current token won't be `Hash`, but it's good to have it here for later in case the parse tag function changes and does some more refined error recovery after `#`.